### PR TITLE
Hide seen-by checkmark for pending messages

### LINKED ISSUE #1470: 'Seen-by' checkmark shows on pending replies
## Description

Reported by @cfm during QA: 
When a message is in a 'pending' state, the 'seen-by' double-checkmark shows, even though the draft has not been sent.

## Steps to Reproduce 
- Send a message and observe seen-by checkmark while message is in pending state
- Can optionally add a `time.sleep(5)` or similar to uploads.py `SendReplyJob.call_api()` to allow more time to inspect the issue  

## Expected Behavior
"Seen-by" checkmark is visible only on successful sending 

## Actual Behavior
"Seen-by" checkmark is visible during pending and successful states

## Comments
I agree that this is confusing (with the double-checkmark in particular). Removing the checkmark(s) until the reply is successfully sent is a simple fix in `ReplyWidget._update_styles()`.

### PR DESCRIPTION
**Pull request by rocodes**
# Description

Fixes #1470
Towards #1465

# Test Plan
- [ ] CI is green
- [ ] Follow Seen-by testing in 0.7.0 release ticket 
- [ ] +1 from @creviera before merge to confirm that we agree on this change

# Checklist

If these changes modify code paths involving cryptography, the opening of files in VMs or network (via the RPC service) traffic, Qubes testing in the staging environment is required. For fine tuning of the graphical user interface, testing in any environment in Qubes is required. Please check as applicable:

 - [ ] I have tested these changes in the appropriate Qubes environment
 - [ ] I do not have an appropriate Qubes OS workstation set up (the reviewer will need to test these changes)
 - [x] These changes should not need testing in Qubes

If these changes add or remove files other than client code, the AppArmor profile may need to be updated. Please check as applicable:

 - [ ] I have updated the [AppArmor profile](https://github.com/freedomofpress/securedrop-client/blob/HEAD/files/usr.bin.securedrop-client)
 - [x] No update to the AppArmor profile is required for these changes
 - [ ] I don't know and would appreciate guidance

If these changes modify the database schema, you should include a database migration. Please check as applicable:

 - [ ] I have written a migration and upgraded a test database based on `main` and confirmed that the migration applies cleanly
 - [ ] I have written a migration but have not upgraded a test database based on `main` and would like the reviewer to do so
 - [ ] I need help writing a database migration
 - [x] No database schema changes are needed

### CODE REVIEW

**Comment by rocodes** [comment]
I know I mentioned wanting a +1 from Allie here - I would settle for a +1 from another maintainer since she is off, or, for just holding off on this until Monday since it's so small. :)  Since Allie did so much work with Erika on this PR, I was wondering if there was any way that the existing behaviour was intentional.

**Review by cfm — APPROVED** [review]
- [x] CI is green
- [x] Follow Seen-by testing in 0.7.0 release ticket

*See below*

- [ ] +1 from @creviera before merge to confirm that we agree on this change

Approving, but deferring for merge to @creviera on Monday, in case this behavior is intended enough that it warrants further discussion.


## Add journalist read receipts ([#1417](https://github.com/freedomofpress/securedrop-client/pull/1417))
- [x] Confirm that when logged in as a user (i.e _journalist_), the tool-tip of the messages and replies on hover displays the authenticated user's username as the default value.
- [x] Confirm that when logged in as a user (i.e _journalist_), the tool-tip of the messages and replies on hover displays the authenticated username **last** if the messages and replies have been read by users other than them.
- [x] Confirm that the username list the tool-tip is initialized to gets updated in a timely manner, in cases where the conversation is read by a new user.
- [x] Confirm that the check-mark is hidden in case the journalist reply fails to deliver.
- [x] Delete a user while logged into the client as a different user and ensure checkmarks are updated as expected

**Review by creviera — APPROVED** [review]
Ah, nice catch. +1

**Pull request merged**
